dev/test/index.ts

```diff
@@ -1073,7 +1073,7 @@ describe('getAll() method', () => {
         expect(actualErrorAttempts).to.deep.eq(expectedErrorAttempts);
       });
     });
-  });
```

Comment:
We normally do `this.timeout(5000);` inside the function, it may be more readable than this, but up to you.

Reply:
That doesn't compile for me:

```
test/watch.ts(865,5): error TS7041: The containing arrow function captures the global value of 'this'.
test/watch.ts(865,10): error TS7017: Element implicitly has an 'any' type because type 'typeof globalThis' has no index signature.
```

If I want to do this, I have to use `function() {..}` somewhere in my test initialization (e.g. in the `describe` call). Rather than to this, I prefer to leave it as is.

Reply:
Yes, we use `function` instead of arrow-style functions if we need to set `this.timeout()`.  That's OK to leave it as is.